sigh/lib/sigh/resign.rb

```diff
@@ -27,7 +27,10 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
       # validate that we have valid values for all these params, we don't need to check signing_identity because `find_signing_identity` will only ever return a valid value
       validate_params(resign_path, ipa, provisioning_profiles)
       entitlements = "-e #{entitlements.shellescape}" if entitlements
-      provisioning_options = provisioning_profiles.map { |fst, snd| "-p #{[fst, snd].compact.map(&:shellescape).join('=')}" }.join(' ')
+      provisioning_options = provisioning_profiles.map do |fst, snd|
```

Comment:
@milch  ok got you - i think it is right so it could be simplified to:

```ruby
provisioning_options = provisioning_profiles.map do |profile|
        profile = File.expand_path(profile).shellescape
        "-p #{profile}"
end.join(' ')
```

i am just concerned about the `join("=")`  - where and why is the `=`  required? (some commandline argument, passing stuff?)

in a sample it works:
```ruby
require "shellwords"
provisioning_profiles = ["a", "b", "~/Downloads/c"]
provisioning_options = provisioning_profiles.map do | profile |
        profile = File.expand_path(profile).shellescape
        "-p #{profile}"
end.join(' ')

puts provisioning_options
```